other/ethhub-cftc-response.md

```diff
@@ -162,11 +162,23 @@ Ethereum has operated uninterrupted for three years as a decentralized blockchai
 
-### 17. How would the introduction of derivative contracts on Ether potentially change or modify the incentive structures that underlie a proof of stake consensus model?
+As exhibit 1 shows, [Add data on volatility of Ether vs. S&P 500 and crude oil] volatility in Ether far outstrips the S&P 500 and even commodities like crude oil. Ether’s volatility is not unique among cryptocurrencies. In fact, as exhibit 2 shows [add data comparing volatility with Ether, Bitcoin, and general crypto market], it’s actually one of the least volatile cryptocurrencies. These dramatic swings in price make it difficult to both determine adequate collateral requirements and acquire said collateral. While these large fluctuations may recede as the cryptocurrency market matures, there’s little concrete evidence that points to this outcome, suggesting that handling volatility will be a key consideration of maintaining collateral in Ether going forward.
 
-Answer:
+Beyond price volatility, Ether also must deal with a collection of exchanges that have limited operating experience, diversification, and liquidity. Five major platforms have the licenses to exchange USD with Ether: Coinbase, Kraken, Bittrex, Gemini, and Circle (operates an OTC market). The oldest of these exchanges is only seven years old (Kraken), and all of them entirely rely on revenue generated from cryptocurrency markets to operate. Furthermore, these exchanges have experienced “flash crashes” which cause forced liquidations for those on margin. On June 21, 2017, the price for Ether on Coinbase went down from $319 to 10 cents in a matter of minutes. Kraken has also faced a flash crash in May 2017, and it now faces a class action lawsuit for misconduct. 
+
+Given the nascent markets for cryptocurrencies and longstanding laws for KYC/AML, these exchanges play a vital role in allowing investors to legally exchange Ether with USD. Even so, until their performance is tested in more market cycles, an unexpected shutdown to their operations via technical problems or business insolvency remains a significant risk to the purchase and holding of Ether.
+
+Granted, one significant caveat to the risks presented above is that investors can easily exchange Ether into other cryptocurrencies via an array of traditional exchanges, decentralized exchanges, and other services. Many of these sites, such as Shapeshift, do not require a login to exchange currencies instantaneously, making such an exchange trivial to most users. Thus, when considering the liquidity of Ether, it’s important to account for the ability to exchange any cryptocurrency (particularly other liquid ones such as Bitcoin) to USD as a secondary source of liquidity. 
```

Comment:
Might be useful to note that Shapeshift has previously complied with the IRS before.

Reply:
Good point. Will look into that. Not familiar with that situation.

Reply:
Do you have a source on this? Couldn't find anything with a cursory look.

Reply:
My mistake. I think I was remembering the IRS requesting documents from Coinbase in 2016 https://support.coinbase.com/customer/portal/articles/2924446-irs-notification

Shapeshift has moved to include KYC/AML though Shapeshift Membership

Reply:
Nice catch. I'll change what I wrote. Unfair to use shapeshift as an example if they're doing the KYC/AML stuff

Reply:
Latest commit edits out Shapeshift entirely and refocuses to volume of ETH/BTC transactions, which I think makes a more pertinent case.